README.md

```diff
@@ -243,8 +243,8 @@ def logistic_predictions(weights, inputs):
 # Training loss is the negative log-likelihood of the training labels.
 def loss(weights, inputs, targets):
     preds = logistic_predictions(weights, inputs)
-    label_probs = preds * targets + (1 - preds) * (1 - targets)
-    return -np.sum(np.log(label_probs))
+    label_probs = np.log(preds) * targets + np.log(1 - preds) * (1 - targets)
```

Comment:
How about `label_logprobs` instead of `label_probs`?

Reply:
You're right, that's better. I will change it.